ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -131,55 +131,34 @@ def getInputSpecification(cls):
       @ Out, specs, InputData.ParameterInput, class to use for specifying input of cls.
     """
     specs = super(GeneticAlgorithm, cls).getInputSpecification()
-    specs.description = r"""The \xmlNode{GeneticAlgorithm} optimizer is a metaheuristic approach
-                            to perform a global search in large design spaces. The methodology rose
-                            from the process of natural selection, and like others in the large class
-                            of the evolutionary algorithms, it utilizes genetic operations such as
-                            selection, crossover, and mutations to avoid being stuck in local minima
-                            and hence facilitates finding the global minima. More information can
-                            be found in:
-                            Holland, John H. "Genetic algorithms." Scientific American 267.1 (1992): 66-73."""
+    specs.description = r"""The \xmlNode{GeneticAlgorithm} is a heuristic optimization technique inspired by the principles 
+                            of natural selection and genetics. Introduced by John Holland in the 1960s, GA mimics the process of 
```

Comment:
Could you revert it back to 'metaheuristic' instead of heuristic? Metaheuristic is a higher level family of algorithms that uses several heuristic techniques, so each specific algorithm in GA can be heuristic but GAs are metaheuristic optimizers. And since our GA contains several of them it should be addressed as metaheuristic.